pkg/cmd/container/top.go

```diff
@@ -203,83 +198,3 @@ func parsePSOutput(output []byte, procs []uint32) (*ContainerTopOKBody, error) {
 	}
 	return procList, nil
 }
-
```

Comment:
You'll need to add a definition for `containerTop` on FreeBSD too please.

If it's identical to the Linux one, you can simply rename the `pkg/cmd/container/top_linux.go` file to `pkg/cmd/container/top_unix.go` and simply add a `!windows` build tag.